### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yashchoudhary8955**
grammatical miskates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ console.log(document.getElementsByTagName("p")[0].innerHTML);
 Interesting features
 --------------------
 
-* Streaming parser: You can pass `parser.parse` an `EventEmitter` and the
+* Streaming parser: You can use this pass `parser.parse` as an `EventEmitter` and the
```
Comment (aredridel) [inline README.md:29]:
```suggestion
* Streaming parser: You can pass an `EventEmitter` to `parser.parse` and the
```

**Review by aredridel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yashchoudhary8955) [reply README.md:29]: yaa this would look better

**Comment by yashchoudhary8955** [comment]
yaa this is also a good option

**Review by aredridel — APPROVED** [review]
Thank you!

**Pull request merged**